### PR TITLE
core[patch],langchain[patch],community[patch]: Bump version dependency of tenacity to >=8.1.0,!=8.4.0,<10

### PR DESCRIPTION
**Pull request by EricDiao**
This should fixes the compatibility issue with graprag as in

- https://github.com/langchain-ai/langchain/discussions/25595

Here are the release notes for tenacity 9 (https://github.com/jd/tenacity/releases/tag/9.0.0)

### CODE REVIEW

**Review by eyurtsev — CHANGES_REQUESTED** [review]
Could you provide an upper pin everywhere? You can specify it like this:  `>=8.1.0,<10`

We're following semver for dependency specification.

It's generally better to have the library fail to install like here: https://github.com/langchain-ai/langchain/discussions/25595

than to fail at run time due to a subtle breaking change

**Comment by eyurtsev** [comment]
Thanks @EricDiao , happy to merge but we need to maintain an upper bound (e.g., <10) to make sure.

If you're doing a bump in a library, it's good to review what the breaking change was. In this case, this looks safe  (https://github.com/jd/tenacity/releases/tag/9.0.0) -- this is good information to include with the PR

File: docs/docs/versions/v0_2/overview.mdx [hunk 1]
```diff
@@ -87,7 +87,7 @@ To understand why we think breaking the dependency of `langchain` on `langchain-
     PyYAML = ">=5.3"
     numpy = "^1"
     aiohttp = "^3.8.3"
-    tenacity = "^8.1.0"
+    tenacity = ">=8.1.0"
```
Comment (eyurtsev) [inline docs/docs/versions/v0_2/overview.mdx:90]:
```suggestion
    tenacity = "^8.1.0"
```

**Review by eyurtsev — APPROVED** [review]

**Comment by eyurtsev** [comment]
Failing unit tests I'll investigate tomorrow

**Comment by EricDiao** [comment]
I am also looking at reproduce the failed tests.

**Comment by EricDiao** [comment]
@eyurtsev I merged changes on master back in https://github.com/langchain-ai/langchain/pull/27201/commits/538e8cb4252878e7f69abe8d99d7e162cfd2fcc4. It works now on my local testbed. You may want to trigger the CI again.

**Comment by eyurtsev** [comment]
Thanks -- had to resolve merge conflicts on community, pushed an update

**Pull request merged**